.circleci/config.yml

```diff
@@ -131,6 +131,9 @@ jobs:
             - oppia/node_modules/
             - oppia/third_party/
             - oppia_tools/
+      - store_artifacts:
```

Comment:
Can we define this on top and reuse in other places?

Reply:
done